_docs/developer/development_instructions/index.md

```diff
@@ -197,6 +197,23 @@ autograding configuration, you'll probably need to:
 
 * [Batch Regrade Homeworks](/instructor/batch_regrade_submissions) already submitted to those gradeables.
 
+---
+
+## Virtual Machine Recovery using Snapshots
+
+In the event of a non-recoverable error while working on Submitty the last resort is to perform a fresh `vagrant up`. However, this process can be time-consuming. To avoid such situations and save time, it is highly recommended to take a snapshot when you first set up your Vagrant environment by following the tutorial links provided below:
```

Comment:
Should be "...while working on Submitty, the last resort is to..." (add a comma)

Reply:
Updated it!